particles/Tutorial/Javascript/Demo/Stores.arcs

```diff
@@ -0,0 +1,44 @@
+import 'GameSchemas.arcs'
```

Comment:
Nit: mostly we call these types of files `data`. It does end up creating Stores, but the name looks weird to me.